app/chat/welcome/page.tsx

```
@@ -0,0 +1,163 @@
+
+      return chatId;
+    } catch (error) {
+      console.error("Error creating welcome chat:", error);
+      // If there's already a welcome chat with this ID (possible with multiple anonymous users),
+      // we need to handle the conflict by using a session-specific ID as fallback
+      if (
+        error instanceof Error &&
+        error.message.includes("Unique constraint")
+      ) {
+        console.log("Welcome chat ID collision, using session-specific ID");
+        const fallbackId = "welcome-" + sessionId.substring(0, 8);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)